agenta-backend/agenta_backend/models/api/api_models.py

```diff
@@ -65,6 +65,14 @@ class CreateAppOutput(BaseModel):
     app_name: str
 
 
+class UpdateApp(CreateApp):
+    pass
+
+
+class UpdateAppOutput(CreateAppOutput):
+    pass
+
```

Comment:
Are these two "passthrough" validation schemas ?
Is there an RFC or some discussion for this ?

Reply:
Yes, `UpdateAppOutput` is a 'passthrough' validation schema that inherits both fields and validation logic from the `CreateAppOutput` model. This means `UpdateAppOutput` will include `app_id` and `app_name`, with the same validations applied to these fields as in `CreateAppOutput`